README.md

```diff
@@ -1,188 +1,181 @@
+1. Open GitKraken
 2. Click `File → Clone Repo`
 3. Select GitHub.com and choose a base folder to clone to.
-4. Write Minecolonies in the Repository to clone and select the one by ldtteam.
+4. Write MineColonies in the "Repository to Clone" and select the one by ldtteam.
 
 ![](https://i.imgur.com/jVTXyCJ.png)
 
+5. Click 'Clone the Repo'.
 
-5. Click Clone the repo.
-
-
-
-#### Compile MineColonies (Command-line)
-1. Navigate to the MineColonies folder, and run a `dir` to make sure the `build.gradle` file is where you expect it to be.
-2. Execute `gradlew build`. This first sets up Forge and downloads the necessary libraries to build MineColonies, be patient as this may take a while.
+#### Compile MineColonies (Command Line)
+1. Navigate to the MineColonies folder and run a `dir` to make sure the `build.gradle` file is where you expect it to be.
+2. Execute `gradlew build`. This first sets up Forge and downloads the necessary libraries to build MineColonies. Be patient, as this may take a while.
     * If you did everything right, `BUILD SUCCESSFUL` will be displayed after it finishes.
     * If you see `BUILD FAILED`, check the error output (it should be right around `BUILD FAILED`), fix everything (if possible), and try again.
 3. Go to `basefolder\MineColonies\build\libs`.
-    *  You should see a `.jar` file named `MineColonies-universal-null.jar`.
-4. Copy the jar into your Minecraft `mods` folder, and you are done! (`~/.minecraft/mods` on Linux)
+    * You should see a `.jar` file named `MineColonies-universal-null.jar`.
+4. Copy the jar into your Minecraft `mods` folder and you are done! (`~/.minecraft/mods` on Linux)
 5. Alternatively, you can also run `./gradlew runClient` to start Minecraft with this jar.
 
 #### Compile MineColonies (Intellij)
-1. Right click the `build.gradle` file, or the background of the folder it's in, and select `Open Folder as Intellij Project`.
-2. Select auto import and make sure a valid gradle and jvm is selected.
+1. Right-click the `build.gradle` file (or the background of the folder it's in) and select `Open Folder as Intellij Project`.
+2. Select `Auto Import` and make sure a valid Gradle and JVM are selected.
 
 ![](https://i.imgur.com/ewccjDZ.png)
 
 3. This will prepare some of the libraries used by Forge.
 4. Click `View → Tool Windows → Gradle`
-5. In the Gradle View, expand the options to `Tasks → fg_runs`
+5. In the Gradle view, expand the options to `Tasks → fg_runs`.
 
 ![](https://i.imgur.com/34H45Tb.png)
 
-6. Click the small refresh symbol in the upper left of the gradle view. This sets up Forge and downloads the necessary libraries to build MineColonies. 
+6. Click the small refresh symbol in the upper left of the Gradle view. This sets up Forge and downloads the necessary libraries to build MineColonies. 
 7. Execute `genIntellijRuns` and restart intellij.
 8. You will see a Minecraft Client and Server startup configuration in the top right, next to the green "start" button.

```

Comment:
```suggestion
8. You will see a runClient and runServer startup configuration in the dropdown at the top right, next to the green "run" button. Select one and press "run".
```

Now step 9 in the line below (145) and the image below it can be safely deleted. You do not need to set a username when testing.

Reply:
This still isn't quite resolved. They're the actual button names in my suggestion.